AbletonMCP_Remote_Script/__init__.py

```
@@ -561,6 +736,314 @@ def _set_tempo(self, tempo):
+        try:
+            self._song.master_track.mixer_device.volume.value = volume
+            
+            result = {
+                "volume": self._song.master_track.mixer_device.volume.value
+            }
+            return result
+        except Exception as e:
+            self.log_message("Error setting master volume: " + str(e))
+            raise
+    
+    def _create_audio_effect_rack(self, track_index, device_index=-1):
```

Ruff (0.14.6)

Unused method argument: `device_index`

(ARG002)